content/pt/docs/7.components-glossary/10.watchquery.md

```diff
@@ -1,39 +1,40 @@
 ---
-title: 'The watchQuery Property'
-description: Watch query strings and execute component methods on change (asyncData, fetch, validate, layout, ...)
-navigation.title: WatchQuery Property
+title: A propriedade watchQuery
+description: Observa as sequência de caracteres de consulta e executa os métodos do componente sobre as mudanças (asyncData, fetch, validate, layout, ...)
+navigation.title: Propriedade WatchQuery
 category: components-glossary
 ---
-# The watchQuery property
+# A propriedade watchQuery
 
 Watch query strings and execute component methods on change (asyncData, fetch, validate, layout, ...)
+Observa as sequência de caracteres de consulta e executa os métodos do componente sobre as mudanças (asyncData, fetch, validate, layout, ...)
 
 ---
 
-- **Type:** `Boolean` or `Array` or `Function` (default: `[]`)
+- **Tipo:** `Boolean` ou `Array` ou `Function` (valor padrão: `[]`)
 
-Use the `watchQuery` key to set up a watcher for query strings. If the defined strings change, all component methods (asyncData, fetch(context), validate, layout, ...) will be called. Watching is disabled by default to improve performance.
+Use a chave `watchQuery` para definir um observador para sequência de caracteres de consulta. Se a sequência de caracteres definida mudar, todos os métodos do componente (asyncData, fetch(context), validate, layout, ...) serão chamados. A observação está desativada por padrão para melhora de desempenho.
 
-If you want to set up a watcher for all query strings, set `watchQuery: true`.
+Se você quiser definir um observador para todas sequências de caracteres de consulta, defina `watchQuery: true`.
```

Comment:
`sequências de caracteres de consulta` what do you mean by `de consulta`?

Reply:
Oh, this specific part corresponds to the piece "query strings", since "strings" are "sequências de caracteres" and "query" is "consulta", I translated "query strings" as "sequências de caracteres de consulta", one has to do with the type of value and other has to do with the purpose of the value. @ChristopheCVB